addon-test-support/@ember/test-helpers/setup-rendering-context.ts

```diff
@@ -84,15 +84,30 @@ export interface RenderOptions {
   Renders the provided template and appends it to the DOM.
 
   @public
-  @param {CompiledTemplate} template the template to render
+  @param {TemplateFactory | unknown} renderable the template or component to render
   @param {RenderOptions} options options hash containing engine owner ({ owner: engineOwner })
   @returns {Promise<void>} resolves when settled
 */
 export function render(
-  template: TemplateFactory,
+  renderable: TemplateFactory | unknown,
   options?: RenderOptions
 ): Promise<void> {
   let context = getContext();
+  let template: TemplateFactory;
+
+  console.log(renderable);
```

Comment:
wip, I was getting a weird error locally about ember-cli-htmlbars not existing 🤔